inline/tests/mod.rs

```diff
@@ -17,6 +20,12 @@ pub struct TestCase {
     out_path: PathBuf,
 }
 
+/// Scans the string with icu_testdata used as provider for Scanner.
+pub fn scan_str(input: &str) -> Vec<Symbol> {
+    let scanner = Scanner::try_new_with_any(icu_testdata::any()).unwrap();
+    scanner.scan_str(input)
+}
+
```

Comment:
This is duplicated in the `core` heading test.
It seems as if we need this more often, so maybe we should move it into the `test_runner` section of `commons`?